modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionListRecordRequestExecutor.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.data.engine.internal.executor;
+
+import com.liferay.data.engine.model.DEDataRecord;
+import com.liferay.data.engine.service.DEDataRecordCollectionListRecordRequest;
+import com.liferay.data.engine.service.DEDataRecordCollectionListRecordResponse;
+import com.liferay.dynamic.data.lists.model.DDLRecord;
+import com.liferay.dynamic.data.lists.service.DDLRecordLocalService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Marcela Cunha
+ */
+public class DEDataRecordCollectionListRecordRequestExecutor {
+
+	public DEDataRecordCollectionListRecordRequestExecutor(
+		DEDataEngineRequestExecutor deDataEngineRequestExecutor,
+		DDLRecordLocalService ddlRecordLocalService) {
+
+		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
+		_ddlRecordLocalService = ddlRecordLocalService;
```

Comment:
sort the assignments 

```suggestion
		_ddlRecordLocalService = ddlRecordLocalService;
		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
```